tests/test_pytypes.py

```diff
@@ -26,6 +26,22 @@ def test_iterator(doc):
     assert doc(m.get_iterator) == "get_iterator() -> Iterator"
 
 
+@pytest.mark.parametrize(
+    "pytype, from_iter_func",
+    [
+        (frozenset, m.get_frozenset_from_iterable),
+        (list, m.get_list_from_iterable),
+        (set, m.get_set_from_iterable),
+        (tuple, m.get_tuple_from_iterable),
+    ],
+)
+def test_from_iterable(capture, doc, pytype, from_iter_func):
```

Comment:
`capture` and `doc` don't appear to be used. Does it still work if you remove these two args?

Reply:
@rwgk yea, I just wanted to be consistent with the rest of the functions in this file. Should I still remove them?

Reply:
Yes I think it's better to remove them. At least that's what I've been doing when adding new tests, only keep what's actually needed.

If you see unused args elsewhere, I wouldn't touch those in this PR, to not distract from the main purpose. (And not enter the slippery slope of feature creep.)